ca/ocsp.go

```diff
@@ -68,6 +68,10 @@ func NewOCSPImpl(
 		issuersByID[issuer.ID()] = issuer
 	}
 
+	if ocspLifetime < 1 || ocspLifetime > 240*time.Hour {
```

Comment:
I think the upper bound should be the more restrictive 168h that the [Microsoft Trusted Root Program](https://learn.microsoft.com/en-us/security/trusted-root/program-requirements#c-revocation-requirements) mandates. The lower bound should be `< 8 * time.Hour` following the BRs and Microsoft Trusted Root Program.

> CAs that issue Server Authentication certificates must support the following OCSP responder requirements:
>
> 2.  Minimum validity of eight (8) hours; Maximum validity of seven (7) days; and